src/Workspaces/Core/Portable/Workspace/Solution/Document.cs

```diff
@@ -200,6 +202,8 @@ public Task<SyntaxTree> GetSyntaxTreeAsync(CancellationToken cancellationToken =
 
         /// <summary>
         /// Gets the root node of the current syntax tree if it is available.
```

Comment:
Yes, to me it is, for the same reason TryGetCompilation was ambiguous (as pointed out by @pilchie earlier). What does it mean to be "available"?